### PR TITLE
Configuring env file and fetching data

### PR DESCRIPTION
**Pull request by Oteng29**
fetched data in fetching-data/index.js and console.log data from home page.

### CODE REVIEW

**Review by OlebogengKhunou — DISMISSED** [review]
Approved

File: next.config.js
Comment (codespace-justin) [inline next.config.js:1]: @Oteng29 Why are the dev env variables the same as the production ones? Just curious i dont see the point in them being there.

**Review by codespace-justin — DISMISSED** [review]
- Work on the naming conventions for functions  such as run and run1
- Add basic comments when referencing or calling functions inside other files where functions are imported to improve code readability and speed up debugging time

**Review by OlebogengKhunou — APPROVED** [review]
approved

**Pull request closed without merging**